packages/build/package.json

```diff
@@ -59,7 +59,7 @@
     "@netlify/plugin-edge-handlers": "^1.11.5",
     "@netlify/plugins-list": "^2.4.2",
     "@netlify/run-utils": "^1.0.6",
-    "@netlify/zip-it-and-ship-it": "^2.7.1",
+    "@netlify/zip-it-and-ship-it": "^3.0.0",
```

Comment:
Should we update in `functions-util` too?

Reply:
Yes, or we can let https://github.com/netlify/build/pull/2434 handle that? I've done it for `build` as part of this PR because otherwise the tests wouldn't pass.

Reply:
Makes sense :+1: